documentcloud/documents/admin.py

```diff
@@ -56,3 +58,30 @@ def save_model(self, request, obj, form, change):
 
     def delete_model(self, request, obj):
         obj.destroy()
+
+
+@admin.register(Entity)
+class EntityAdmin(admin.ModelAdmin):
+    """Entity Admin"""
+
+    list_display = ("name", "kind", "mid", "description", "wikipedia_url")
+    list_filter = ("kind", "description")
+    search_fields = ("name", "kind", "mid", "description", "wikipedia_url")
+    show_full_result_count = False
+    paginator = LargeTablePaginator
+    ordering = ("pk",)
+    fields = ("name", "kind", "mid", "description", "wikipedia_url")
+
+    readonly_fields = ()
+
+    @transaction.atomic
+    def save_model(self, request, obj, form, change):
+        super().save(request, obj, form, change)
+        transaction.on_commit(
+            lambda: solr_index.delay(
+                obj.pk, field_updates={f: "set" for f in form.changed_data}
+            )
+        )
+
+    def delete_model(self, request, obj):
```

Comment:
Documents require special clean up on deletion which the document `destroy` method handles.  Entity's have no such requirement and do not have a `destroy` method.  Leaving `delete_model` undefined so it uses the correct parent implementation.